packages/fxa-react/components/Footer/index.test.tsx

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import Footer from '.';
+
+describe('Footer', () => {
+  it('renders as expected', () => {
+    render(<Footer />);
+
+    const linkMozilla = screen.getByTestId('link-mozilla');
+
+    expect(linkMozilla).toHaveAttribute(
+      'href',
+      'https://www.mozilla.org/about/?utm_source=firefox-accounts&utm_medium=Referral'
+    );
+    expect(linkMozilla.firstElementChild).toHaveAttribute('role', 'img');
```

Comment:
I'm 100% cool with this approach, but I'm curious if this was intentional over using test ID on the child elements? Is it better in some way, or just different?

Reply:
Eh, I could have used a testID here, I just wanted to make sure that the logo was inside of the link.

Reply:
Works for me